test/unit/test-error.js

```diff
@@ -755,10 +755,9 @@ describes.sandboxed('reportError', {}, (env) => {
   });
 });
 
-describe.configure().run('detectJsEngineFromStack', () => {
+describe('detectJsEngineFromStack', () => {
```

Comment:
It looks like this test suite is specific to a quirk of SauceLabs: `"does not masquerade with UserAgent"` -- is it worth keeping something to that effect? Are these tests not running at all after this change?

Reply:
If you're referring to the change from `describe.configure().run` to `describe`, it's a no-op. (`configure()` is needed only when there is a special case, like `.skipSafari()` for example.) The tests inside this block will continue to run in their specific cases.